### DISCUSSION TITLE
Umami on AWS

### DISCUSSION THREAD

**Discussion by Vultik** [umami-software/umami#770] [Q&A]
Hello,
Thank you for Umami.
Do you know if it's possible to deploy umami on AWS Amplify ? For now, i use heroku but i have other apps on aws.
I don't want to use an EC2 instance that seems to overkill.

**Comment by lalit2005** [comment]
I don't believe you can host the backend and database on AWS Amplify. @mikecao might be the right person to ask

If you would like to host Umami for free, I would prefer to go with frontend on Vercel and postgres on Supabase. Both of them have generous free tiers too.

And here's a nice tutorial too → https://dev.to/jakobbouchard/setting-up-umami-with-vercel-and-supabase-3a73

**Reply by Vultik** [reply]
The database is hosted on Amazon RDS
I'm sure that is possible to do it on Amplify since it's made for this purpose : having a static frontend made with framework like Next.js and configure also connection for the backend. But i think major modifications should me made in the umami code itself.
Now, the build is failling at SRS (Server-side Rendering) with the @sls-next/serverless-component module.

**Reply by Vultik** [reply]
Otherwise, it could be also implemented with API Gateway + Lambda + Cloudfront/S3, but once again major modifications should be made on umami code.
